api/nodes/serializers.py

```diff
@@ -646,10 +646,13 @@ def create(self, validated_data):
         tag_instances = []
         affiliated_institutions = None
         region_id = None
+        license_details = None
         if 'affiliated_institutions' in validated_data:
             affiliated_institutions = validated_data.pop('affiliated_institutions')
         if 'region_id' in validated_data:
             region_id = validated_data.pop('region_id')
+        if 'license_type' in validated_data or 'license' in validated_data:
```

Comment:
I would explore modifying `get_license_details` on this page so it works whether a node is passed in or not.  It returns a dictionary of the format 
```
return {
        'id': license_id,
        'year': license_year,
        'copyrightHolders': license_holders,
    }
```

which is extracted from validated_data.  You might have to also pop `license_type` and `license` off of validated_data.